src/_utils.ts

```diff
@@ -9,6 +9,14 @@ export function toBufferLike(val: any): BufferLike {
     return val;
   }
 
+  if (
+    Buffer.isBuffer(val) ||
```

Comment:
`Buffer.from([]) instanceof Uint8Array` is true. We can avoid Node.js dependency (`Buffer`)